### DISCUSSION TITLE
Predict - results of games show - NO RESULTS

### DISCUSSION THREAD

**Discussion by JackHipeek** [betfinio/public#50] [Bugs and Reports]
<img width="970" alt="Snímek obrazovky 2024-10-04 v 0 10 36" src="https://github.com/user-attachments/assets/db2e782a-046d-4c8f-a6a9-17e84bb9b4e1">

**Comment by bf-monster** [comment]
fixing now

**Comment by bf-monster** [comment]
fixed, should be working now
